docs/fsharp/tour.md

```diff
@@ -1,4 +1,158 @@
 # Tour of F# #
 
+The best way to learn about F# is to read and write F# code.  This tour of some of the features of the language will highlight some things and give you some code snippets that you can execute on your machine.  To learn about setting up a development environment, check out [Getting Started](tutorials/getting-started/index.md).
+
+There are two primary concepts in F#: functions and types.  This tour will emphasize features of the language which fall into these two concepts.
+
+## How to Run the Code Samples
+
+>![NOTE]
+Eventually, this documentation site will have the ability to run these code samples directly in your browser.
+
+The quickest way to run these code samples is to use [F# Interactive](tutorials/fsharp-interactive/index.md).  Just copy/paste the code samples and run them there.  Alternatively you can set up a project to compile and run the code as a Console Application in [Visual Studio](tutorials/getting-started/getting-started-visual-studio.md) or [Visual Studio Code and Ionide](tutorials/getting-started/getting-started-vscode.md).
+
+## Functions and Modules
+
+The most fundamental pieces of any F# program are Functions organized into Modules.  [Functions](language-reference/functions/index.md) perform work on inputs to produce outputs, and they are organized under Modules, which are the primary way you group things in F#.  They are defined using the [`let` binding](language-reference/functions/let-bindings.md), which give the function a name and define its arguments.
+
+[!code-fsharp[BasicFunctions](../../samples/snippets/fsharp/tour.fs#L25-L51)]
+
+`let` bindings are also how you bind a value to a name, similar to a variable in other languages.  However, unlike language such as C#, these values are immutable by default.  The lines between values and functions are a bit blurred, and that's intentional.  More on that later.
```

Comment:
I'd suggest not directly contrasting with C# here. Yes, C# variables are mutable by default. So are JavaScript variables, C++ variables, and so on. 

I'd also define the terms ***mutable*** and ***immutable***, because this is introductory material.